spaces/S000088/properties/P000033.md

```diff
@@ -0,0 +1,16 @@
+---
+space: S000088
+property: P000033
+value: true
+---
+
+Consider open subspaces $A_n=q([0,\omega_1)\times[0,\omega]\times\{2n,2n+1\})$
+and $B_n=q([0,\omega_1]\times[0,\omega)\times\{2n,2n-1\})$ for $n\in\mathbb Z$.
+Each of them is {P33}
+(for $A_n$ is countably compact similarily as {S35|P19}
+and $B_n$ is Hausdorff and {P17} [(Explore)](https://topology.pi-base.org/spaces?q=%24%5Csigma%24-compact%2Bt2%2B%7Emetacompact))
```

Comment:
```suggestion
(for $A_n$: use the fact that the product of a {P19} space with a {P16} space is {P19};
for $B_n: because it is Hausdorff and {P17}
[(Explore)](https://topology.pi-base.org/spaces?q=%24%5Csigma%24-compact%2Bt2%2B%7Emetacompact))
```
I think this explains more that the previous `{S35|P19}` without having to get into details.